communities.json

```
@@ -771,6 +771,12 @@
       "quote": "Discord has been amazing for accelerating the development of React Spring and React Three Fiber. Screen sharing and instant communication have been pivotal in resolving bugs and coordinating with our contributors.",
       "inviteCode": "poimandres",
       "githubUrl": "https://github.com/pmndrs"
+    },
+    {
+      "logo": "multimc.svg",
+      "title": "MultiMC",
+      "inviteCode": "0k2zsXGNHs0fE4Wm",
```

Do you want to update the invite code here to your vanity invite code?
```suggestion
      "inviteCode": "multimc",
```